internal/cmd/testdata/TestKeysListCmd/list_all

```diff
@@ -1,4 +1,4 @@
   NAME        ISSUED FOR  CREATED       EXPIRES           EXTENSION DEADLINE  
   front-door  admin       24 hours ago  never             never               
-  side-door   admin       24 hours ago  6 hours from now  tomorrow            
+  side-door   admin       24 hours ago  6 hours from now  26 hours from now   
```

Comment:
I guess this was another bug with `go-prettytime`. 26 hours from now should not be "tomorrow" since it's more than 24 hours.